bolt/CMakeLists.txt

```diff
@@ -121,7 +123,7 @@ option(BOLT_BUILD_TOOLS
   "Build the BOLT tools. If OFF, just generate build targets." ON)
 
 add_custom_target(bolt)
-set_target_properties(bolt PROPERTIES FOLDER "BOLT")
+set_target_properties(bolt PROPERTIES FOLDER "BOLT/Meta")
```

Comment:
This probably should be something more neutral then a company name.

Reply:
This is not supposed to be the company name, but meta in the sense of [self-referential](https://en.wikipedia.org/wiki/Meta_(prefix)). That is, targets that only exists to trigger the build of other targets. In this case, all of BOLT (as in `make all` if BOLT was its own build system). "Metatarget" would disambiguate the meaning.

An alternative could be "[Phony](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)".